runtime/doc/news.txt

```diff
@@ -149,6 +149,7 @@ API
   escape sequences to the terminal when Nvim is running in the |TUI|.
 • |nvim_echo()| can set the |ui-messages| kind with which to emit the message.
 • |nvim_echo()| can create |Progress| messages
+• |nvim_open_win()| can show a 'statusline'. see |status-line| for details.
```

Comment:
```suggestion
• |nvim_open_win()| floating windows can show a 'statusline'. Plugins can use
  `style='minimal'` or `:setlocal statusline=` to hide the statusline.
```

Reply:
Currently it is not controlled by style; it only depends on the floating window local statusline option. Since the default value is empty, floating statuslines will not appear. It is rendered only when this value is non-empty and laststatus is set to 1 or 2. Adding style control would require updating many existing float_spec tests.

Reply:
yes, that is the correct behavior of style=minimal.

however, we could save that for a separate PR if it's complicated. but it is important, because style=minimal exists for that purpose.

Reply:
resolved and added test case for style.